src/form-sections/amount.ts

```diff
@@ -277,7 +277,7 @@ export class MGCEditPlanAmount extends LitElement {
                   }
 
                   // eslint-disable-next-line no-param-reassign
-                  (iaButton as IauxButton).isDisabled = true;
+                  (iaButton as MGCButton).isDisabled = true;
```

Comment:
Q: Do we need this `iaButton as MGCButton` since the callback signature already casts it as an MGCButton `iaButton: MGCButton`?